### PR TITLE
Bookmarks, some refactor and README update

### PR DESCRIPTION
**Pull request by Samyak2**
# Description

This branch contains some major changes:

 - Bookmarks feature: from #70 
 - Renamed `Outputs.entries` to `Outputs.histories` (#70)
 - Add features section in README
 - Re-formatted using [black](https://github.com/psf/black)
 - Some API changes
 - Documentation update for these changes

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] I have read the [contribution guidelines](https://browser-history.readthedocs.io/en/latest/contributing.html) and followed it as far as possible. 
- [x] I have performed a self-review of my own code (if applicable)
- [x] I have commented my code, particularly in hard-to-understand areas (if applicable)
- [x] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings 
- [x] Any dependent and pending changes have been merged and published

### CODE REVIEW

**Review by ObaraEmmanuel — COMMENTED** [review]
Some of the linting seem a bit excessive

File: tests/test_browsers.py [hunk 1]
```diff
@@ -2,106 +2,233 @@
...
+    assert his[0] == (
+        datetime.datetime(
+            2020,
+            8,
+            3,
+            0,
+            29,
+            4,
+            tzinfo=datetime.timezone(datetime.timedelta(seconds=19800), "IST"),
+        ),
```
Comment (ObaraEmmanuel) [inline tests/test_browsers.py:30]: This linting seems excessive.

Comment (Samyak2) [reply tests/test_browsers.py:30]: I agree. This formatting was done by black ([1](https://github.com/psf/black/), [2](http://www.locallyoptimal.com/blog/2019/08/23/why-you-should-use-black-for-your-python-style-linting/)) and so we didn't have much control over what it did.

I tried to fix it manually but black seems to not like it :laughing:

Comment (ObaraEmmanuel) [reply tests/test_browsers.py:30]: I guess black knows best then 😂😂😂

**Pull request merged**